wallet/lib/ipc/ipc_engine.dart

```diff
@@ -133,25 +134,29 @@ class IPCEngine {
   }
 
   Future<void> _handleEaselLink(String link) async {
-    final queryParameters = Uri.parse(link).queryParameters;
+    bool isHome = false;
+    final currentWallets = walletsStore.getWallets().value;
+
+    isHome = isHomeRoute();
+    if (!repository.getBool(key: kISUserCreatingAccount) && isHome == false) {
```

Comment:
kISUserCreatingAccount should be kIsUserCreatingAccount

Reply:
And also why we are saving this.

Reply:
We have to take care of this thing because when a user clicks on an NFT and doesn't have an account created yet after the creation of an account it is called pop and removes all the routes and then goes to the NFT. So that is why we are saving it